### ISSUE TITLE
[Feature Request] Code to use trained model for Entity Linking

### ISSUE THREAD

**Issue by dungtn** [izuna385/Zero-Shot-Entity-Linking#4]
Hi @izuna385,

Great work! I want to use your codebase for a downstream task. I find it a bit hard to modify the code to run prediction on given sentences. Can you please help?

Best!

**Comment by izuna385** [comment]
Currently I'm developing more user-friendly linking tool but yes PRs are welcome!
Thanks

**Comment by ruanchaves** [comment]
I just built a simple Flask server around a slightly modified version of this repository. You can take a look at [my fork](https://github.com/ruanchaves/Zero-Shot-Entity-Linking) ( specially the `src/client.py` and `src/api.py` files ).

I won't submit a PR as it looks like a hack, but maybe it can give you some ideas.

**Comment by JLUGQQ** [comment]
> I just built a simple Flask server around a slightly modified version of this repository. You can take a look at [my fork](https://github.com/ruanchaves/Zero-Shot-Entity-Linking) ( specially the `src/client.py` and `src/api.py` files ).
> 
> I won't submit a PR as it looks like a hack, but maybe it can give you some ideas.

Maybe the source data was changed. I run your code and your code had a error that "FileNotFoundError: [Errno 2] No such file or directory: './data/documents/wikipedia.json'" when I run the preprocess.py

**Comment by ruanchaves** [comment]
@JLUGQQ You're right, somebody deleted the wikipedia.json from the file, maybe because it was eating up too much space on their Drive.   

Anyway, I have a backup of the original source data, you can download it here: [zeshel.tar.bz2](https://drive.google.com/file/d/1ioosODuKU0lhAzs96bDC1PHwaS03l5AO/view?usp=sharing)
